lib/src/base-styles/_nx-colors.scss

```diff
@@ -92,6 +92,21 @@
   --nx-color-counter: var(--nx-swatch-grey-05);
   --nx-color-counter-active-text: var(--nx-swatch-white);
 
+  //NxProgressBar
+  --nx-color-progress-bar-background: var(--nx-swatch-indigo-95);
+  --nx-color-progress-bar-progress: var(--nx-swatch-teal-60);
+  --nx-color-progress-bar-text-success: var(--nx-swatch-green-25);
+  --nx-color-progress-bar-icon-success: var(--nx-color-progress-bar-text-success);
+  --nx-color-progress-bar-text-error: var(--nx-swatch-red-40);
```

Comment:
use `--nx-color-validation-invalid-message` instead

Reply:
> use `--nx-color-validation-invalid-message` instead

I think `--nx-color-alert-error` is more fitting in this case. The variable's name is closer to `--nx-color-progress-bar-text-error` while being the same color swatch. I committed this update here: [966ad55](https://github.com/sonatype/sonatype-react-shared-components/pull/1049/commits/966ad55dc2c7852fdaa1bb29830b579cf4a7d99f)